CODEOWNERS

```diff
@@ -4,90 +4,77 @@
+/west.yml                                 @nrfconnect/ncs-code-owners
 /west-test.yml                            @thst-nordic
 
-# CI specific west
-/test-manifests/99-default-test-nrf.yml     @thst-nordic
-
-# Github Actions
+# Dot folders
 /.github/                                 @nrfconnect/ncs-ci
 /.github/test-spec.yml                    @nrfconnect/ncs-test-leads
-
-# Quarantine for the CI and Twister
-/scripts/quarantine.yaml                  @nrfconnect/ncs-test-leads
-
-# VS Code Configuration files
 /.vscode/                                 @FilipZajdel
 
 # Applications
+/applications/				  @nrfconnect/ncs-code-owners
 /applications/asset_tracker_v2/           @nrfconnect/ncs-cia @coderbyheart
 /applications/connectivity_bridge/        @nrfconnect/ncs-cia @nordic-auko
 /applications/ipc_radio/                  @dchat-nordic
 /applications/machine_learning/           @pdunaj
 /applications/matter_bridge/              @nrfconnect/ncs-matter
 /applications/matter_weather_station/     @nrfconnect/ncs-matter
-/applications/nrf_desktop/                @MarekPieta
 /applications/nrf5340_audio/              @nrfconnect/ncs-audio
-/applications/serial_lte_modem/           @SeppoTakalo @MarkusLassila @rlubos @tomi-font
+/applications/nrf_desktop/                @MarekPieta
+/applications/serial_lte_modem/           @nrfconnect/ncs-co-networking @SeppoTakalo @MarkusLassila @tomi-font
 /applications/zigbee_weather_station/     @milewr
+/applications/**/*.rst                    @nrfconnect/ncs-doc-owners
+
 # Boards
-/boards/                                  @anangl
-/boards/nordic/thingy91*                  @anangl @nrfconnect/ncs-cia
+/boards/                                  @nrfconnect/ncs-co-boards
+/boards/nordic/thingy91*                  @nrfconnect/ncs-co-boards @nrfconnect/ncs-cia
+
 # All cmake related files
-/cmake/                                   @tejlmand
-/CMakeLists.txt                           @tejlmand
-# All Kconfig related files
-Kconfig*                                  @tejlmand
-# Sysbuild related files
-/sysbuild/                                @tejlmand @nordicjm
-/cmake/sysbuild/                          @tejlmand @nordicjm
+/cmake/                                   @nrfconnect/ncs-co-build-system
+
 # All doc related files
-/doc/_extensions/                         @gmarull
-/doc/_scripts/                            @carlescufi
-/doc/_static/                             @carlescufi
-/doc/_utils/                              @gmarull
-/doc/cache.yml                            @gmarull
-/doc/CMakeLists.txt                       @carlescufi
-/doc/**/conf.py                           @carlescufi
-/doc/kconfig/                             @gmarull
-/doc/nrf/                                 @carlescufi
+/doc/					  @nrfconnect/ncs-co-doc
+/doc/tags.yml                             @nrfconnect/ncs-co-doc @umapraseeda @b-gent
+/doc/**/*.rst				  @nrfconnect/ncs-doc-owners
```

Comment:
It will request review from all tech writers (each tech writer will get email notification; at least in default notification settings), but review from only one tech writer satisfies the requirement and would allow the PR to be merged (if all other required teams have approved).

Reply:
Should each NCS team have a subteam for doc where the tech-writer that works with the team would be added? Or does this sound like an overkill?

It looks that we would need to have the generic /doc/**/*.rst at top, and then the specific entries (either with individual tech writes or the subteams) below. If the specific entries catch all .rst files, then the generic won't hit and therefore only the individuals will be notified.

Is there any chance to organize the doc directory with NCS teams subdirectories? Or would such change result in horrible mess?

Reply:
> Should each NCS team have a subteam for doc where the tech-writer that works with the team would be added? Or does this sound like an overkill?

So far, we've always worked based on the assumption that each team knows the tech writers working with them and knows when to add them ([source 1](https://nordicsemi.atlassian.net/wiki/spaces/NCS/pages/108225083/Documentation+process#Documentationprocess-Sampleandapplicationdocumentation) , [source 2](https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120302385/NCS+team+assignment) ). So from this perspective, the proposed changes sound like an overkill.

You really need tech writers in the codeowners file when one needs to monitor changes across different areas. For example, one of the reasons I asked to be added because I'm working with changes to the doc structure, so I need a cross-team overview to general sections.

Maybe a better option than having 13 tech writers in doc-owners is to have some of them in the group (3? 4?), and this group would ensure that the relevant tech writers are added to the PRs individually. But I still feel like we're reinventing the wheel for a process that is working relatively well (teams adding tech writers based on the need).

> Is there any chance to organize the doc directory with NCS teams subdirectories? Or would such change result in horrible mess?

The doc directory in sdk-nrf? No. A tech writer on Matter could contribute to apps/matter_bridge, protocols/matter, samples/matter, which would result in a spaghetti-style horrible mess of duplicated directories.

Reply:
@greg-fer can we perhaps start with this, and then refine it as we go? Because we will discover a bunch of things the moment we merge this and enable required reviews by code owners.

Reply:
Apologies, didn't want to block this. I trust your judgment, @carlescufi . There was also @b-gent who had concerns though, and I know from some colleagues in TechPubs that they were also pulled into PRs out of the blue and had no idea why.